### DISCUSSION TITLE
NZ:P VR

### DISCUSSION THREAD

**Discussion by Cybrock9000** [nzp-team/nzportable#855] [General]
Is it possible to port NZP to VR, i did find a open source quake vr project that you could build off of: https://github.com/vittorioromeo/quakevr/tree/master

**Comment by MotoLegacy** [comment]
Because of the long term hassle that is porting this project around and supporting new ports, I'm currently not looking to add something new. Feel free to tackle the challenge yourself though!

**Comment by Cybrock9000** [comment]
Thanks for responding so quickly and I think I will take the challenge
